releases/go/.git-cliff.toml

```
@@ -0,0 +1,21 @@
+[git]
+tag_pattern = "^releases/go/mpl/v*"
+filter_commits = true
+commit_parsers = [
+    { message = '^feat\((go|all languages)\)', group = "Features"},
+    { message = '^fix\((go|all languages)\)', group = "Fixes"},
+    { message = '^chore\((go|all languages)\)', group = "Maintenance"},
+    { message = '^docs\((go|all languages)\)', group = "Maintenance"},
+    { message = '^revert\((go|all languages)\)', group = "Fixes"},
+    { message = '^style\((go|all languages)\)', group = "Refactor"},
```

```suggestion
    { message = '^style\((go|all languages)\)', group = "Miscellaneous"},
```

Just for consistency with https://github.com/aws/aws-cryptographic-material-providers-library/blob/1c563bdf8985f4c79b376929faa61496964dc69c/.releaserc.cjs#L99